lib/client/mfa.go

```diff
@@ -50,6 +50,10 @@ func (p *mfaPrompt) PromptPIN() (string, error) {
 
 // PromptMFAChallengeOpts groups optional settings for PromptMFAChallenge.
 type PromptMFAChallengeOpts struct {
+	// BeforePrompt is an optional method to print before the prompt.
```

Comment:
Typo, it was supposed to be "an optional message". I can change it to a callback since that's more flexible though. Maybe call it "AdviceBefore" like emacs lisp terminology lol